sonata-project/admin-bundle/3.30/config/packages/sonata_admin.yaml

```diff
@@ -0,0 +1,2 @@
+sonata_admin:
+    title: 'Sonata Admin'
```

Comment:
With this configuration an exception will be thrown during a rendering of template [Core/dashboard.html.twig](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/views/Core/dashboard.html.twig#L88)

> The block type "sonata.admin.block.admin_list" does not exist

To avoid this error message, i use this configuration:

```yaml
sonata_admin:
    dashboard:
        blocks:
          - { type: sonata.admin.block.admin_list, position: left }

sonata_block:
    blocks:
        sonata.admin.block.admin_list:
            contexts: [admin]
```

Reply:
i guess only the `sonata_block` part is needed

Reply:
This should be done here: https://github.com/symfony/recipes-contrib/pull/235

Reply:
@OskarStark enabling `sonata.admin.block.admin_list` block should be in recipe for AdminBundle, not for BlockBundle, because this block is a part of AdminBundle

Reply:
You are right, my fault"